components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -1,233 +1,199 @@
+                    >
+                        {{ specialty }}
+                    </option>
+                </select>
+                <label
+                    for="locales"
+                    class="my-2 text-primary-text text-sm font-bold font-sans"
+                >
+                    {{ $t("modHealthcareProfessionalSection.selectLocales") }}
+                </label>
+                <select
+                    id="locales"
+                    v-model="sectionFields.healthcareProfessionalLocales"
+                    data-testid="mod-healthcare-professional-section-spoken-locales"
+                    name="Locales"
+                    multiple
+                    class="mb-5 px-3 py-3.5 w-96 h-32 bg-secondary-bg rounded-lg border border-primary-text-muted
+                                text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+                >
+                    <option
+                        v-for="(locale, index) in localeList"
+                        :key="`${locale}-${index}`"
+                        :value="locale"
+                    >
+                        {{ locale }}
+                    </option>
+                </select>
+            </div>
         </div>
     </div>
 </template>
 
 <script lang="ts" setup>
-import { onMounted, reactive, ref } from 'vue'
+import { nextTick, onBeforeMount, onMounted, type Ref, ref, watch } from 'vue'
+import { useRoute } from 'vue-router'
+import { type ToastInterface, useToast } from 'vue-toastification'
 import { ModerationScreen, useModerationScreenStore } from '~/stores/moderationScreenStore'
+import { useHealthcareProfessionalsStore } from '~/stores/healthcareProfessionalsStore'
 import { Locale, type LocalizedNameInput, Insurance, Degree, Specialty } from '~/typedefs/gqlTypes'
 import { multiSelectWithoutKeyboard } from '~/utils/multiSelectWithoutKeyboard'
-import SVGTrashCan from '~/assets/icons/trash-can.svg'
-import SVGProfileIcon from '~/assets/icons/profile-icon.svg'
+import { useI18n } from '#imports'
 
-const healthcareProfessionalSectionFields = reactive({
-    healthcareProfessionalNameArray: [] as Array<LocalizedNameInput>,
-    localizedFirstName: '',
-    localizedLastName: '',
-    localizedMiddleName: '',
-    nameLocale: Locale.EnUs,
-    healthcareProfessionalAcceptedInsurances: [] as Array<Insurance>,
-    healthcareProfessionalDegrees: [] as Array<Degree>,
-    healthcareProfessionalSpecialties: [] as Array<Specialty>,
-    healthcareProfessionalLocales: [] as Array<Locale>
-})
+let toast: ToastInterface
+
+const route = useRoute()
+
+const { t } = useI18n()
 
 const moderationScreenStore = useModerationScreenStore()
-const professionalNameArray = healthcareProfessionalSectionFields.healthcareProfessionalNameArray
+const healthcareProfessionalsStore = useHealthcareProfessionalsStore()
+const sectionFields = healthcareProfessionalsStore.healthcareProfessionalSectionFields
```

Comment:
can this just be `healthcareProfessional`, `healthcareProfessionalFields`, or `healthcareProfessionalModel`? 
Any of these work for me :)